cmd/server-test/timeout_analyzer_test.go

```diff
@@ -54,13 +51,9 @@ func (suite *TimeoutErrorAnalyzerIntegrationSuite) TearDownTest() {
 func (suite *TimeoutErrorAnalyzerIntegrationSuite) TestAnalyzerTimeoutErr() {
 	suite.sendEvent(successJSON)
 
-	suite.GrepTrue(suite.r, suite.errMessage)
+	suite.GrepTrue(suite.r, `msg="analysis failed: timeout exceeded, try increasing analyzer_review in config.yml" analyzer=Dummy app=lookoutd error="rpc error: code = DeadlineExceeded desc = context deadline exceeded"`)
```

Comment:
@carlosms @se7entyse7en The config file could be a different file if passed with `--config=`
I'd purpose `/config.yml/the config file/s`, or similar, to cover those cases.